src/plugins/plot/stackedPlot/StackedPlotItem.vue

```diff
@@ -145,19 +187,73 @@ export default {
             this.status = status;
             this.updateComponentProp('status', status);
         },
+        setSelection() {
+            let childContext = {};
+            childContext.item = this.object;
+            this.context = childContext;
+            if (this.removeSelectable) {
+                this.removeSelectable();
+            }
+
+            this.removeSelectable = this.openmct.selection.selectable(
+                this.$el, this.context);
+        },
         loadingUpdated(loaded) {
             this.loading = loaded;
             this.updateComponentProp('loading', loaded);
         },
         getProps() {
             return {
+                limitLineLabels: this.showLimitLineLabels,
                 gridLines: this.gridLines,
                 cursorGuide: this.cursorGuide,
                 plotTickWidth: this.plotTickWidth,
                 loading: this.loading,
                 options: this.options,
-                status: this.status
+                status: this.status,
+                colorPalette: this.colorPalette
             };
+        },
+        getPlotObject() {
+            if (this.object.configuration && this.object.configuration.series) {
+                //If the object can be persisted, allow initialization of the config from it's persisted config
+                return this.object;
+            } else {
+                // If the  object cannot be persisted, initialize the series config with the persisted config from the stacked plot
+                const configId = this.openmct.objects.makeKeyString(this.object.identifier);
+                let config = configStore.get(configId);
+                if (!config) {
+                    const persistedConfig = this.domainObject.configuration.series.find((seriesConfig) => {
```

Comment:
`this.domainObject` is the parent stacked plot? vs `this.object` which is the telemetry object? Maybe we could improve the naming of these objects to make this a little clearer?

Reply:
Yes, `this.object` is any object that's part of the stacked plot's composition - a telemetry object or an overlay object.

Reply:
Hmm I can't think of a better variable name since this could be a telemetry object OR an overlay object. :)

Reply:
childObject? I'm just trying to differentiate it from `this.domainObject` somehow